apps/src/applab/debugArea.js

```diff
@@ -0,0 +1,236 @@
+/* jshint
+ funcscope: true,
+ newcap: true,
+ nonew: true,
+ shadow: false,
+ unused: true,
+
+ maxlen: 90,
+ maxparams: 3,
+ maxstatements: 200
+ */
+/* global $ */
+
+var applabMsg = require('./locale');
+var dom = require('../dom');
+var utils = require('../utils');
+
+var debugArea = module.exports;
+
+/** @type {jQuery} */
+var rootDiv_ = null;
```

Comment:
to make this comment not entirely theoretical, I will point out that we have run into some bugs in the past where we have collisions of global variables when someone forgets the word "var".